roles/rsyslog/README.md

```diff
@@ -21,53 +46,56 @@ localhost ansible_user=YOUR_ANSIBLE_USER openshift_logging_use_ops=True
 localhost ansible_user=YOUR_ANSIBLE_USER openshift_logging_use_ops=True
 ```
 
-2. vars.yaml stores variables which are passed to ansible to control the tasks.
+vars.yaml
+---------
+1. vars.yaml stores variables which are passed to ansible to control the tasks.
 
-   Currently, this rsyslog roles support 2 deployment variables, rsyslog__viaq and rsyslog__example.  I.e., there are 3 sets of deployment - rsyslog__viaq: true, rsyslog__example: true, and both false.  Theoretically, both true could be set and the specified configuration files are deployed, but rsyslog does not work properly with the configuration.
+   Currently, this rsyslog roles support 2 deployment variables, rsyslog_viaq and rsyslog_example.  I.e., there are 3 sets of deployment - rsyslog_viaq: true, rsyslog_example: true, and both false.  Theoretically, both true could be set and the specified configuration files are deployed, but rsyslog does not work properly with the configuration.
 
    See the variables section for each variable.
 
    Sample vars.yaml file for the viaq case.
 ```
-rsyslog__enabled: true
+rsyslog_enabled: true
 # install viaq packages & config files
-rsyslog__viaq: true
-rsyslog__capabilities: [ 'viaq' ]
-rsyslog__group: root
-rsyslog__user: root
+rsyslog_viaq: true
+rsyslog_capabilities: [ 'viaq' ]
+rsyslog_group: root
+rsyslog_user: root
 elasticsearch_server_host: es_hostname
```

Comment:
Need to update here to the rsyslog_elasticsearch parameter

rsyslog_elasticsearch:
  - name: default
    server_host: es_hostname
    server_port: 9200
    ca_cert: "{{rsyslog_viaq_config_dir}}/es-ca.crt"
    cert: "{{rsyslog_viaq_config_dir}}/es-cert.pem"
    key: "{{rsyslog_viaq_config_dir}}/es-key.pem"

Reply:
Using the loop index allows us to get rid of the `name` field for the `default` value, to know how to set up the `if`/`else if`/`else` conditions.

But - rsyslog actions have a `name` field https://www.rsyslog.com/doc/v8-stable/configuration/actions.html#general-action-parameters - we will need to use this for stats and rsyslog monitoring e.g. if you have several elasticsearch outputs, you will need to name them, so you can look at a prometheus/grafana dashboard and tell that elasticsearch-ops is having problems but not elasticsearch-apps.

Reply:
Where is it used in the code to reflect that? @richm

Reply:
@sradco something like this:
```
action(
  type="omelasticsearch"
  name="elasticsearch-apps"
...
```